modules/http4s/src/main/scala/com/evolutiongaming/smetrics/Http4sMetricsOps.scala

```diff
@@ -0,0 +1,116 @@
+package com.evolutiongaming.smetrics
+
+import cats.effect._
+import cats.implicits._
+import com.evolutiongaming.smetrics.MetricsHelper._
+import org.http4s.metrics.TerminationType.{Abnormal, Error, Timeout}
+import org.http4s.metrics.{MetricsOps, TerminationType}
+import org.http4s.{Method, Status}
+
+object Http4sMetricsOps {
+
+  def of[F[_] : Sync](
```

Comment:
why do you need Sync ?

Reply:
it needs for flatMap operations, but it can be replaced by Applicative and FlatMap
https://github.com/iodkowski/smetrics/blob/http4s/modules/http4s/src/main/scala/com/evolutiongaming/smetrics/Http4sMetricsOps.scala#L34
https://github.com/iodkowski/smetrics/blob/http4s/modules/http4s/src/main/scala/com/evolutiongaming/smetrics/Http4sMetricsOps.scala#L61

Reply:
please use `Monad`

Reply:
replaced by Monad